app/server/src/server/framework-preset-server.ts

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Configuration } from 'webpack';
+
+export function webpack(config: Configuration) {
+  return {
+    ...config,
+    module: {
+      ...config.module,
+      rules: [
+        ...config.module.rules,
+        {
+          test: /\.html$/,
+          use: [
+            {
+              loader: require.resolve('html-loader'),
```

Comment:
What do we need this for?

Reply:
entirely copy pasta from @storybook/html. I was unclear for the need of them there. We can nuke right?

Reply:
I think so. This is to allow you to import `.html` files in storybook builds that use this addon. That's not something you would do with `@storybook/server`, right?